odoo/cli/server.py

```diff
@@ -136,6 +138,11 @@ def main(args):
         for db_name in preload:
             try:
                 odoo.service.db._create_empty_database(db_name)
+            except ProgrammingError as err:
+                if err.pgcode == errorcodes.INSUFFICIENT_PRIVILEGE:
+                    _logger.warning("Could not determine if database exists: %s" % err)
```

Comment:
Hmm I see, now it makes more sense ;-) Perhaps an info is not so bad then, since others could run into this too.